streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -325,7 +324,6 @@ public void run() {
 
         try {
             runLoop();
-            cleanRun = true;
```

Comment:
Just discussed offline with @mjsax , his previous commit has leaked a change that is removing its usage here. We will add this variable back in another PR.